packages/sit-onyx/src/components/examples/GridPlayground/GridElement/GridElement.vue

```diff
@@ -0,0 +1,74 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+import type { GridElementConfig } from "../EditGridElementDialog/EditGridElementDialog.vue";
+
+const props = defineProps<
+  GridElementConfig & {
+    label: string;
+    mode?: "default" | "outline";
+  }
+>();
+
+const emit = defineEmits<{
+  click: [];
+}>();
+
+const gridClasses = computed(() => {
+  return [
+    `onyx-grid-span-${props.columnCount}`,
+    ...Object.entries(props.breakpoints ?? {}).map(([breakpoint, columns]) => {
+      return `onyx-grid-${breakpoint}-span-${columns}`;
+    }),
+  ];
+});
+</script>
+
+<template>
+  <button
+    class="grid-element"
+    :class="[...gridClasses, props.mode === 'outline' ? 'grid-element--outline' : '']"
+    type="button"
+    :aria-label="props.label"
```

Comment:
you might want to give the visual people a chance to read the label too :)

```suggestion
    :aria-label="props.label"
    :title="props.label"
```

Reply:
I intended the label only to be used for screen readers / Playwright. Do you think there is a benefit for showing "Grid element X" to the users?

Reply:
it's fine, it's a demo ^^ I just thought about what jonathan once said when he added the title everywhere where only aria-label exists, but I'm not sure we kept htat pattern consistent